plugins/woocommerce/includes/rest-api/Controllers/Version3/class-wc-rest-product-variations-controller.php

```diff
@@ -614,7 +614,7 @@ public function get_item_schema() {
 				'status'                => array(
 					'description' => __( 'Variation status.', 'woocommerce' ),
 					'type'        => 'string',
-					'default'     => ProductStatus::PUBLISH,
+					'default'     => 'publish', // Not using ProductStatus constants here due the class not being loaded upon installation.
```

Comment:
```suggestion
					// Not using ProductStatus constants here due the class not being loaded upon installation.
					// https://github.com/woocommerce/woocommerce/issues/37464
					'default'     => 'publish',
```

A bit of a nit, and this is entirely optional, but linking to the specific issue in question might be useful if at some future point we want to address this and need additional context.